controllers.err.js

```diff
@@ -1,3 +1,14 @@
 exports.notFound = (req, res, next) => {
     res.status(404).send({msg : 'Path not found'});
-}
\ No newline at end of file
+}
+
+exports.errorHandler = (err, req, res, next) => {
+    
+    if (err.code === "22P02") {
+      res.status(400).send({ msg: "Bad Request" });
+    } else if (err.message === "undefined") {
+      res.status(err.status).send({ msg: err.message });
+    } else {
+      res.sendStatus(500);

```

Comment:
A note on error handling: 

I want you to consider extracting your error handling out by 'category' e.g. PSQL Errors, custom errors, 500 error
This isn't a hard bit of feedback, if you feel better doing it this way I'm okay with that. But generally for readability it's nice to split errors by functionality